src/modules/endpoint/endpoints/__tests__/categories.spec.js

```diff
@@ -8,104 +8,78 @@ jest.unmock('../categories')
 describe('categories', () => {
   const categoriesJSON = [{
     id: 3650,
-    permalink: {
-      url_page: 'anlaufstellen'
-    },
+    path: '/augsburg/de/anlaufstellen',
     title: 'Anlaufstellen zu sonstigen Themen',
     status: 'publish',
     content: '',
-    parent: 0,
+    parent: {},
     order: 75,
     available_languages: {
-      en: 4361,
-      ar: 4367,
-      fa: 4368
+      en: {path: '/augsburg/en/anlaufstellen'}
     },
     thumbnail: 'https://cms.integreat-ap…/03/Hotline-150x150.png',
     modified_gmt: '2017-01-01'
   },
   {
     id: 3649,
-    permalink: {
-      url_page: 'willkommen'
-    },
+    path: '/augsburg/de/willkommen',
     title: 'Willkommen',
     status: 'publish',
     content: '',
-    parent: 0,
+    parent: {},
```

Comment:
why do we need an object here? 🤔

Reply:
This is the json returned by the api v3, which returns the id, the full cms url and the path for each parent, so parent is an object.

Reply:
Ah I see... But maybe you could still fill them up to real parent objects like we get them from cms (for categories with no parent we get`{id: 0,  path: null, url: null }`)

Reply:
okay, sure :)